documentation/IDTA-01001/modules/ROOT/pages/includes/IDTA-01001_ValueOnly.adoc

```diff
@@ -410,15 +607,14 @@ For a BasicEventElement named “MyBasicEvent”, the payload is minimized to th
         },
         {
           "type": "Property",
-          "value": "CurrentValue"
+          "value": "MaxRotation"
         }
       ]
     }
-  }
 }
 ----
 
-===== JSON-Schema for the ValueOnly-Serialization
+==== JSON-Schema for the ValueOnly-Serialization
```

Comment:
The JSON Schema (which was not changed, i.e. I cannot make a comment on that during review) includes elements that are not defined in the meta model/part 1, namely `OperationRequestValueOnly` and `OperationResultValueOnly`.
These should not be part of this document but there should be a section in part 2 adding to this JSON schema for classes added in part 2 meta model.

Reply:
true